dev_setup.sh

```diff
@@ -638,15 +594,5 @@ chmod +x bin/mycroft-say-to
 chmod +x bin/mycroft-skill-testrunner
 chmod +x bin/mycroft-speak
 
-# create and set permissions for logging
-if [[ ! -w /var/log/mycroft/ ]] ; then
-    # Creating and setting permissions
-    echo 'Creating /var/log/mycroft/ directory'
-    if [[ ! -d /var/log/mycroft/ ]] ; then
-        $SUDO mkdir /var/log/mycroft/
-    fi
-    $SUDO chmod 777 /var/log/mycroft/
-fi
```

Comment:
Do we need to create the new directories - looks like this gets done in `bin/mycroft-start` but wouldn't if people are following the `dev_setup.sh` > `start-mycroft.sh` path?

Reply:
Seems I missed that part! It should also log to `$XDG_STATE_HOME/mycroft`, changed it!